resources/upgrading-to-aep/v4-analytics-to-aep-analytics.md

```diff
@@ -0,0 +1,315 @@
+| trackLocation:data: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/geo_poi.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/geo_poi.html)) | This functionality is available in the [Places extension](../../using-mobile-extensions/adobe-places). |
+| trackBeacon:Data: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/ibeacon.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/beacon.html)) | Support modified, [see guide](https://aep-sdks.gitbook.io/docs/resources/user-guides/track-beacon) |
+| trackingClearCurrentBeacon ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/ibeacon.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/beacon.html)) | Support modified, [see guide](https://aep-sdks.gitbook.io/docs/resources/user-guides/track-beacon) |
+| trackLifetimeValueIncrease:data: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/lifetime_value.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/lifetime_value.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionStart: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionUpdate: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionEnd: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+| trackTimedActionExists: ([iOS](https://marketing.adobe.com/resources/help/en_US/mobile/ios/timed_actions.html), [Android](https://marketing.adobe.com/resources/help/en_US/mobile/android/timed_actions.html)) | This functionality can be recreated using the [Analytics](../../using-mobile-extensions/adobe-analytics) and [User Profile](../../using-mobile-extensions/profile) extensions. |
+
+#### API Changes
```

Comment:
Below is more like initializing or setup for the extension.  Not really API change